src/main/scala/com/fulcrumgenomics/sv/FgSvDef.scala

```diff
@@ -0,0 +1,16 @@
+package com.fulcrumgenomics.sv
+
+import com.fulcrumgenomics.FgBioDef.FilePath
+import htsjdk.samtools.util.OverlapDetector
+import htsjdk.tribble.AbstractFeatureReader
+import htsjdk.tribble.bed.{BEDCodec, BEDFeature}
+
+object FgSvDef {
+  /** Builds an [[OverlapDetector]] frpom a BED file */
+  def overlapDetectorFrom(bed: FilePath): OverlapDetector[BEDFeature] = {
```

Comment:
Is there a reason to have this in a `*Def`?  Why not just `Intervals.overlapDetectorFromBed()`?